dashboard/main_heatmap.py

```diff
@@ -34,51 +34,73 @@
 METRIC_HISTORY_TABLE_NAME = os.environ['METRIC_HISTORY_TABLE_NAME']
 
 JOB_STATUS_QUERY = f"""
-SELECT
-  x.test_name,
-  x.job_status,
-  SAFE_CAST(DATE(timestamp, 'US/Pacific') AS STRING) AS run_date,
-  x.stackdriver_logs_link AS logs_link,
-  x.logs_download_command AS logs_download_command,
-  x.kubernetes_workload_link AS workload_link,
-  x.uuid
-FROM (
-  SELECT
+SELECT 
+  test_name,
+  job_status,
+  SAFE_CAST(DATE(timestamp, 'US/Pacific') AS STRING) as run_date,
+  logs_link,
+  workload_link,
+  uuid
+FROM
+(SELECT 
```

Comment:
Nit: you may want to keep the query format consistent in those files. i.e. location of `(`.